app/tests/integration/nimbus/pages/browser/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""Browser Model"""
+
+
+class Browser:
+    def execute_script(selenium, *args, script=None, context=None):
+        if "chrome" in context:
```

Comment:
Python Q: Is `context` a string, or array/object? I see one usage below which seems to specify `"chrome"` exact string, but curious why the `in` instead of exact string matching here.

Reply:
```context``` is a string. Using the ```in``` operator, python will compare the strings.

Reply:
Ah! OK, so it looks like `in` does substring checking (so `context` could be "chrome" or "somethinglikechrome").